### PR TITLE
added a few style guide examples

### PR DESCRIPTION
**Pull request by PatrickAlphaC**

### CODE REVIEW

File: contracts/style.md [hunk 1]
```diff
@@ -28,10 +46,66 @@ Our starting point is the [official Solidity Style Guide](https://solidity.readt
...
+
+```javascript
+uint private s_myVar;
+uint256 private immutable myImmutVar;
+uint256 private constant s_myConstVar;
+
+function getMyVar_() external view returns(uint256){
+    return s_myVar;
+}
+```
```
Comment (se3000) [inline contracts/style.md:73]: Github's suggestion feature isn't working well with the markdown code blocks inside of markdown comment, so I delimited with...

I'd suggest changing the bad example to something like:
```
function getMyVar_(uint _multiplier) external view returns(uint256){
    return _mutliplier * s_myVar;
```
and updating the good version to mirror it. Reason being I think the `uint` in the function signature is especially disruptive, so possibly worth demonstrating. Used the extra parameter to also demonstrate the discouraged leading underscore on params.

**Review by se3000 — APPROVED** [review]
Optional suggestion.

File: contracts/style.md (hunk 1 above)
Comment (PatrickAlphaC) [reply contracts/style.md]: Updated! What you think?

**Comment by PatrickAlphaC** [comment]
Ok, merging! Thank you!

**Pull request merged**